src/components/SudokuGrid/Row.js

```diff
@@ -1,20 +1,14 @@
 import Cell from "./Cell";
-import { v4 as uuidv4 } from "uuid";
 
-export default function Row({
-  currentSudokuNumbers,
-  rowNumber,
-  mask,
-  onCellInRowClick,
-}) {
+export default function Row({ rowData, rowNumber, mask, onCellInRowClick }) {
   return (
     <div className="Sudoku__row">
-      {currentSudokuNumbers[rowNumber].map((number, columnNumber) => {
+      {rowData.map((cell, columnNumber) => {
         const isEditable = mask[rowNumber][columnNumber] !== 0 ? false : true;
```

Comment:
This could be simplified.
`mask[rowNumber][colNumber] !== 0` evaluates already to a boolean. That means that you could say:

```js
const isEditable = mask[rowNumber][columnNumber] === 0;
```

`isEditable` will be true if that comparison evaluates to true, and false otherwise